aea/connections/scaffold/connection.yaml

```diff
@@ -2,6 +2,7 @@ name: scaffold
 author: fetchai
 version: 0.1.0
 license: Apache-2.0
+aea_version: ">=0.2.2"
```

Comment:
In my opinion it should be `==0.2.2` since we are still having breaking changes between versions. We should aim to move to `>=0.2.2, <0.3.0`. But I think `>=0.2.2` is wrong.

Reply:
Applies to all files!